mmpose/models/backbones/resnet.py

```diff
@@ -85,13 +85,16 @@ def __init__(self,
 
     @property
     def norm1(self):
+        """nn.Module: the normalization layer named "norm1" """
```

Comment:
The meaning of this is not clear enough. Does it mean "Get the name of normalization layer "norm1" "?